cloud/azure/bin/lib/azure.sh

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env bash
+
+#######################################
+# Create resource group
+# Arguments:
+#   1: The resource group name 
+#   2: The location of the resource group
+#######################################
+function azure::create_resource_group() {
+  if [ $(az group exists --name "${1}") = false ]; then
```

Comment:
I don't know the exact output of this command, but I think == "false" is preferred here for a string match